### PR TITLE
Hotfix animations

### PR DESCRIPTION
**Pull request by ahmed-shehata**
Small fix for transitioningDelegate (weak var)

### CODE REVIEW

**Comment by ahmed-shehata** [comment]
👍

**Comment by haldun** [comment]
👍

**Comment by ahmed-shehata** [comment]
👍

File: AtlasUI/AtlasUI/AtlasCheckout.swift [hunk 1]
```diff
@@ -41,9 +41,9 @@ public struct AtlasCheckout {
             }
 
             let sizeSelectionViewController = SizeSelectionViewController(checkout: self, sku: sku)
-
+            let checkoutTransitioning = CheckoutTransitioningDelegate()
             let navigationController = UINavigationController(rootViewController: sizeSelectionViewController)
-            navigationController.transitioningDelegate = CheckoutTransitioningDelegate()
+            navigationController.transitioningDelegate = checkoutTransitioning
```
Comment (bonkey) [inline AtlasUI/AtlasUI/AtlasCheckout.swift]: @ahmed-shehata Was it really a problem? If so, I'm super curious why it does work, when the other didn't?

**Pull request merged**